reference/promise-types/vars.markdown

```diff
@@ -378,3 +378,144 @@ two_example_com::
 ```
 
 (Promises within the same bundle are evaluated top to bottom, so vars promises further down in a bundle can overwrite previous values of a variable. See [**normal ordering**][Normal Ordering] for more information).
+
+## Defining variables in foreign bundles
+
+As a general rule, variables can only be defined from within the bundle where a
+variable is defined. There are a few notable exceptions to this general rule
```

Comment:
The first sentence doesn't quite make sense.

Reply:
Updated